lib/local_authorities_importer.rb

```diff
@@ -0,0 +1,63 @@
+require 'ostruct'
+
+class LocalAuthoritiesImporter
+  include JsonDownloader
+
+  LOCAL_AUTHORITY_TYPES = "DIS,LBO,MTD,UTA,COI,LGD"
+
+  def self.import_from_mapit
+    new.authorities_from_mapit
+  end
+
+  def authorities_from_mapit
+    url = "#{MAPIT_BASE_URL}areas/#{LOCAL_AUTHORITY_TYPES}"
+    mapit_las = get_mapit_authorities(url)
+
+    mapit_las.each do |mapit_la|
```

Comment:
Possibly nitpicking: Could you rename the `get_mapit_authorities` method to just `mapit_authorities` and then call `each` on that directly? I'm also wondering whether the `url` variable could become a class constant/method, so that you don't have to pass it as a parameter.